Packs/Logzio/Integrations/Logzio/Logzio.py

```diff
@@ -247,6 +262,9 @@ def main():
                              ' Logz.io Security API token, or both.')
         region = demisto.params().get('region')
         first_fetch_time = demisto.params().get('fetch_time', '1 hours')
+        filter_tags = demisto.params().get('tags', None)
```

Comment:
the `tags` parameter was not added to the yml integration configuration, as far as I can tell.